modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx

```diff
@@ -130,7 +130,8 @@ export function MaxLengthProperties({
 						onChange={({target: {value}}) =>
 							onSettingsChange({
 								name: 'maxLength',
-								value: value && Number(value),
+								value:
+									Number(value) < 280 ? Number(value) : 280,
```

Comment:
Hey @Artartur, good suggestion, the only thing that concerns me is, if the user is typing for example "123", if he types any more digits, the number will be forced to be 280, I don't think this should be the behavior, I think we should keep "123 " in this case. To do this, instead of putting 280 if the condition is not valid, we can put `settings.maxLength`, which is the current value in the data structure

```suggestion
								value:
									Number(value) <= 280
										? Number(value)
										: (settings.maxLength as number),
```

Reply:
I see, It makes sense. I've done all the requests. thx @danvb112!